daq_utils.py

```diff
@@ -32,6 +32,11 @@
 CAMERA_ANGLE_BEAM = 0 # viewing angle is in line with beam, upstream from the sample, facing downstream, top toward ceiling
 CAMERA_ANGLE_ABOVE = 1 # viewing angle is directly above sample facing downward, top of view is downstream
 CAMERA_ANGLE_BELOW = 2 # viewing angle is directly below sample facing upward, top of view is downstream
+global mag1ViewAngle, mag2ViewAngle, mag3VivewAngle, mag4ViewAngle
+mag1ViewAngle = CAMERA_ANGLE_BEAM
+mag2ViewAngle = CAMERA_ANGLE_BEAM
+mag3ViewAngle = CAMERA_ANGLE_BEAM
+mag4ViewAngle = CAMERA_ANGLE_BEAM
```

Comment:
These shouldn't all be CAMERA_ANGLE_BEAM, should they?

Reply:
this is a good set of defaults, that they are all along the beam, like on AMX/FMX. this also won't require any additional changes on AMX/FMX